packages/core/src/input/Input.css

```diff
@@ -178,6 +184,29 @@
   border-bottom: var(--input-borderWidth) var(--input-borderStyle) var(--input-borderColor);
 }
 
+/* Style applied if `bordered={true}` */
+.saltInput.saltInput-bordered {
+  --input-border: var(--salt-size-border) var(--salt-container-borderStyle) var(--input-borderColor);
+  --input-borderWidth: 0;
```

Comment:
`input-border` vs `input-borderWidth`.

First one has border width defined as well? I'd assume one of them is for activation indicator?

Reply:
yes. and the second one is set to 0 to negate the  border-bottom being applied from default case. instead of that i can just do `border-bottom: none; ` to avoid confusion

Reply:
Although not ideal, this is the same approach as existing multiline input, so will treat it the same way.